solidity/random-beacon/.yarnrc

```diff
@@ -0,0 +1 @@
+"@keep-network:registry" "https://registry.yarnpkg.com"
```

Comment:
Don't we want to publish the package to the default npmjs.com registry as we do for other packages?

Reply:
I changed it because of the errors described [here](https://www.flowdock.com/app/cardforcoin/tech/threads/wXfCDfJtOz0ZmfYWJgY_g7Zahq7)
I looked at the cov-pool where we started using `yarn` and it resolves to `https://registry.yarnpkg.com` Basically, npmjs.com registry also would work, however since we're using `yarn`, don't we want to stay with `yarn` registry?

Reply:
We publish all packages to the default registry (npmjs.com). It [seems like the yarn registry uses the npm registry anyways](https://stackoverflow.com/questions/58071109/difference-between-yarn-registry-and-npm-registry/58071376).

The [`.npmrc` file](https://github.com/keep-network/keep-core/blob/main/.npmrc) seems obsolete and should be removed, as we no longer publish packages to GH registry.

Reply:
Let's remove both `.npmrc` and `.yarnrc` files. We resolve to default registry just fine. If we need a custom registry then we can add it later 9fe8e2a